src/components/StickerTemplate.js

```diff
@@ -20,37 +20,41 @@ import defensa from "../../assets/appAssets/test/defensa.png";
 import medioCentro from "../../assets/appAssets/test/medioCentro.png";
 import arquero from "../../assets/appAssets/test/arquero.png";
 
+// se borra luego esto es para tamaños
+import jugador from "../../assets/appAssets/test/ksa/ksa_5.png"
+
 const pos = '../assets/appAssets/test/'
 const { width } = Dimensions.get('window')
 
+/* cambiar luego de subir a online SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS */
+import Constants from 'expo-constants';
+/* SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS */
```

Comment:
```suggestion
import Constants from 'expo-constants';
```
Ya estamos en staging y este PR va a master, que falta para llegar a "online" xP?

Reply:
Ya se arreglo en otro commit